services/Skyra.Notifications/SubscriptionManager.cs

```diff
@@ -0,0 +1,175 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Timers;
+using AngleSharp;
+using AngleSharp.Html.Dom;
+using Microsoft.Extensions.Logging;
+using Skyra.Database;
+using Skyra.Shared.Results;
+
+namespace Skyra.Notifications
+{
+	public class SubscriptionManager
+	{
+		private readonly IBrowsingContext _browsingContext;
+		private readonly IDatabase _database;
+
+		private readonly ILogger<SubscriptionManager> _logger;
+		private readonly PubSubClient _pubSubClient;
+		private readonly Timer _resubTimer;
+		private HttpClient _httpClient;
+
+		private JsonSerializerOptions _jsonSerializerOptions = new JsonSerializerOptions
+		{
+			PropertyNamingPolicy = JsonNamingPolicy.CamelCase
+		};
+
+		private Dictionary<string, DateTime> _resubscribeTimes;
```

Comment:
Initialize this

Reply:
[it is](https://github.com/skyra-project/skyra/pull/1910/files/e9dd02d1dc93bc5b5bd9acb3cb54cf7fc5c33e2e#diff-b92397fa1a889e64810aa7d57ee54ebb60803ec3040aebd51102cabddea12856R71)